test/Event.test.ts

```diff
@@ -0,0 +1,36 @@
+import fetchMock from "jest-fetch-mock";
```

Comment:
Node.js SDK implementation should adhere to these specs in this repo:https://github.com/FeatureProbe/server-sdk-specification